config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -167,6 +167,17 @@ config BT_DEVICE_NAME_GATT_WRITABLE
 config CHIP_NFC_COMMISSIONING
     default y
 
+# Enable Factory Data support by default
+
+config CHIP_FACTORY_DATA
```

Comment:
automatic factory data merge is enabled too?

Reply:
yes, it is enabled by default when CHIP_FACTORY_DATA_BUILD is set to y.